pkg/operator/status.go

```diff
@@ -250,15 +263,24 @@ func (c *Controller) computeStatusConditions(tuned *tunedv1.Tuned, conditions []
 
 				copyAvailableCondition()
 			}
-			reportDegraded = anyProfileDegraded(profileList)
+			reportProfileDegraded = anyProfileDegraded(profileList)
+			reportProfileProgressing = anyProfileProgressing(profileList)
 		} else {
-			// Profile supplied, check if Degraded should be reported based on its status conditions
-			reportDegraded = profileDegraded(profile)
+			// Profile supplied, check if ProfileDegraded should be reported based on its status conditions
```

Comment:
I now this is really my original comment, but could you adjust this for me (as you're changing the line) please: `s/status conditions/status conditions./` . Thank you.